src/Sulu/Bundle/WebsocketBundle/Controller/FallbackController.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/*
+ * This file is part of the Sulu CMF.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\WebsocketBundle\Controller;
+
+use Sulu\Bundle\WebsocketBundle\Connection\DummyConnection;
+use Sulu\Component\Websocket\AppManagerInterface;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Provides fallback interface for websocket apps
+ */
+class FallbackController
+{
+    /**
+     * @var AppManagerInterface
+     */
+    private $appManager;
+
+    function __construct(AppManagerInterface $appManager)
+    {
+        $this->appManager = $appManager;
+    }
+
+    /**
+     * Redirect message to app and returns value as json response
+     * @param string $appName
+     * @param Request $request
+     * @return JsonResponse
+     */
+    public function send($appName, Request $request)
+    {
+        $sessionName = ini_get('session.name');
+
+        $message = $request->get('message');
+        $id = $request->cookies->get($sessionName);
```

Comment:
i have copied the code from symfony sessions ... i use this unique key to identify the user! this key is the same for the whole session and i can connect which websocket connection with a ajax connection ...